src/conditional-flow/string-conditions.js

```diff
@@ -49,7 +68,21 @@ function getMiddleLetter (val1) {
 // Winter - December to February
 function seasonForMonth (monthName) {
 
-  // TODO: write code in this function body to pass the tests
+if (monthName === "December" || monthName === "January" || monthName === "February") {
```

Comment:
There might be a cleaner way of doing this. Perhaps using arrays to store the months that belong to certain seasons. For example `const winterMonths = ['December', 'January', 'February']` :thinking:

Reply:
Wouldn't you then have to iterate through the arrays to check? which could over complicate?

Reply:
Nope! Array's have a few helpful methods associated with them that makes it easy to check for primitive values such as strings. You could do something along the lines of 
```js
if(winterMonths.includes(monthName)) {
   return "winter"
}
```